packages/amplication-server/src/core/auth/auth.controller.ts

```diff
@@ -182,6 +183,13 @@ export class AuthController {
       isNew = false;
```

Comment:
Discuss if a new event is needed to detect user uses switches between different login types (github or sso or email+password)